FastSimulation/BaseParticlePropagator/src/BaseParticlePropagator.cc

```diff
@@ -659,13 +660,13 @@ BaseParticlePropagator::propagateToBeamCylinder(const XYZTLorentzVector& v, doub
 
   // Here are the four possible solutions for
   // 1) The trajectory radius
-  std::vector<double> helixRs(4,static_cast<double>(0.));
+  std::array<double,4> helixRs = {{0.}};
   helixRs[0] = (b - std::sqrt(b*b - a*c))/(2.*a); 
   helixRs[1] = (b + std::sqrt(b*b - a*c))/(2.*a); 
   helixRs[2] = -helixRs[0]; 
   helixRs[3] = -helixRs[1]; 
   // 2) The azimuthal direction at the second point
-  std::vector<double> helixPhis(4,static_cast<double>(0.));
+  std::array<double,4> helixPhis = {{0.}};
```

Comment:
This works correctly, but what it says is initialize the first element to 0.0 and then independent of that value-initialize the rest (which sets them to 0). So if you did std::array<double,4> helixPhis = {{1.}}; you would get a 1 followed by 3 zeroes which might not be what is expected ...
```C++
std::array<double,4> helixPhis{};
```
seems more clear.